go.mod

```diff
@@ -0,0 +1,64 @@
+module github.com/edgexfoundry/device-onvif-camera
+
+require (
+	github.com/IOTechSystems/onvif v0.0.2-0.20220301065030-7cf2dd734897
```

Comment:
@jpwhitemn , We'll need to get this new module TSC approved.

Reply:
This library is originally forked from https://github.com/use-go/onvif which has some bugs and additions needed for ONVIF support.  Stats below are from the original source project/repository
Total increase in new imports:  4 (github.com/beevik/etree v1.1.0, github.com/elgs/gostrgen v0.0.0-20161222160715-9d61ae07eeae, github.com/gin-gonic/gin v1.6.2, github.com/gofrs/uuid v3.2.0+incompatible).  These are already in use:  github.com/stretchr/testify, golang.org/x/net

Release tags:  1
Contributors: 7
License: MIT
Stars:  157
Forks: 83
Watchers: 6
Importers: 15 (via godoc metrics)

Obviously this code has some issues since IOTech members had to fork and fix/update the code a bit.  The original project does appear to still be active (last update 21 days ago), but not very much activity over the past year.
IOTech plans to maintain the fork and update as the original dictates.